app/src/main/java/uk/gov/onelogin/MainActivity.kt

```diff
@@ -36,7 +36,11 @@ class MainActivity : AppCompatActivity() {
             LaunchedEffect(key1 = Unit) {
                 viewModel.apply {
                     next.observe(lifecycleOwner) {
-                        navController.navigate(it)
+                        navController.navigate(it) {
+                            popUpTo(navController.graph.id) {
```

Comment:
what does this do exactly? 🤔

Reply:
essentially clears the back stack. So this navigation is always off either splash screen or sign in screen so think we want to clear them out the back stack

Reply:
and we want to do this every single time we handle a navigation?

It might work for now, but let's keep an eye on it.